lib/packwerk/commands/check_command.rb

```diff
@@ -0,0 +1,48 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Packwerk
+  module Commands
+    class CheckCommand < BaseCommand
+      extend T::Sig
+      include UsesParseRun
+
+      sig { override.returns(T::Boolean) }
+      def run
+        if @files_for_processing.files.empty?
+          @out.puts(<<~MSG.squish)
+            No files found or given.
+            Specify files or check the include and exclude glob in the config file.
+          MSG
+
+          true
```

Comment:
It feels like it should return `false` if the above was an error, but this is the existing behaviour.

Also, we pass an `err_out` around but it feels like we're not using it as much as we should...

In the spirit of keeping this PR a low impact refactor, I decided not to change those.